### PR TITLE
Removed arm-navigation package, added moveit support

### PR DESCRIPTION
**Pull request by shaun-edwards**
This pull request addresses github/ros_industrial_issues#15.  The arm navigation packages have been removed.  Moveit support/config packages have been added in accordance with ROS-I directory structure.  The `motoman_config` package remains for backwards compatibility.  It has been marked for deprecation and should be remove post hydro.

@JeremyZoss, please review.

### CODE REVIEW

File: motoman_driver/launch/robot_state_fs100.launch [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!--
+  Contoller specific version of 'robot_interface.launch'.
+
+
+  Usage:
+    robot_interface_streaming_fs100.launch robot_ip:=<value>
+-->
```
Comment (gavanderhoorn) [inline motoman_driver/launch/robot_state_fs100.launch:7]: Same here

File: motoman_driver/launch/robot_state_dx100.launch [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
+  Contoller specific version of 'robot_interface.launch'.
+
+
+  Usage:
+    robot_interface_streaming_dx100.launch robot_ip:=<value>
```
Comment (JeremyZoss) [inline motoman_driver/launch/robot_state_dx100.launch]: This should be robot_state_dx100.launch

File: motoman_driver/launch/robot_state_fs100.launch [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!--
+  Contoller specific version of 'robot_interface.launch'.
+
+
+  Usage:
+    robot_interface_streaming_fs100.launch robot_ip:=<value>
```
Comment (JeremyZoss) [inline motoman_driver/launch/robot_state_fs100.launch]: this should be robot_state_fs100.launch

File: motoman_mh5_support/launch/robot_interface_streaming_mh5.launch [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  Defaults provided for mh5:
+   - 7 joints
+
+  Usage:
+    robot_interface_streaming_mh5.launch robot_ip:=<value> controller:=<fs100|dx100>
+-->
+<launch>
+	<arg name="robot_ip" />
+	
+  <!-- controller: Controller name (fs100 or dx100) -->
```
Comment (JeremyZoss) [inline motoman_mh5_support/launch/robot_interface_streaming_mh5.launch:14]: fix indent (and in other motoman___support/launch/robot_interface_streaming__.launch files)

File: motoman_mh5_support/launch/robot_state_visualize_mh5.launch [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  Defaults provided for mh5:
+   - 7 joints
+
+  Usage:
+    robot_state_visualize_sia20d.launch robot_ip:=<value> controller:=<fs100|dx100>
+-->
+<launch>
+	<arg name="robot_ip" />
+
+  <!-- controller: Controller name (fs100 or dx100) -->
```
Comment (JeremyZoss) [inline motoman_mh5_support/launch/robot_state_visualize_mh5.launch:14]: fix indent (and in other motoman_support/launch/robot_state_visualize_.launch files)

File: motoman_sia20d_moveit_config/.setup_assistant [hunk 6]
```diff
@@ -0,0 +1,8 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: motoman_sia20d_support
+    relative_path: urdf/sia20d.urdf
+  SRDF:
+    relative_path: config/sia20d.srdf
```
Comment (JeremyZoss) [inline motoman_sia20d_moveit_config/.setup_assistant]: update to match new SRDF path: config/motoman_sia20d.srdf

**Comment by JeremyZoss** [comment]
Should delete `motoman_sia20d_moveit_config / default_warehouse_mongo_db` (and add to `.gitignore`?)

**Comment by JeremyZoss** [comment]
+1

**Comment by shaun-edwards** [comment]
@gavanderhoorn - Your comment got deleted, but I think the answer is that at the lowest level the fs100/dx100 state/interface launch files call the lowest level launch files with the byte-swapping parameter.  The fs and dx controllers have the same driver node, but different endianess (like the fanuc controller and sim).  The reason for the separate launch files is a legacy from when the drivers were in different packages entirely.

**Pull request merged**